CCS compatibility for failure stores as available from 9.4.0 (#6381)
elastic/docs-content: manage-data/data-store/data-streams/failure-store.md

## Cross Cluster Search compatibility [ccs-compatibility]

```{applies_to}
stack: ga 9.4
```

> :::{important}

Reviewer: we should probably take this out of the important block

> :::{important}

Reviewer 2: ```suggestion

```

> Accessing the failure store across clusters using `::failures` is not yet supported.

Reviewer 2: ```suggestion
Accessing the failure store across clusters using `::failures` is only supported starting with {{es}} 9.4.
```

> :::

Reviewer 2: ```suggestion

```
